qemu/tests/qemu_guest_agent.py

```diff
@@ -117,15 +116,13 @@ def _check_ga_service(self, session, cmd_check_status):
         return s == 0
 
     @error_context.context_aware
-    def gagent_install(self, session, vm, *args):
-        if args and isinstance(args, tuple):
-            gagent_install_cmd = args[0]
-        else:
-            self.test.error("Missing config 'gagent_install_cmd'")
-
-        if not gagent_install_cmd:
-            self.test.error("Gagent_install_cmd's value is empty.")
-
+    def gagent_install(self, session, vm, gagent_install_cmd):
+        """
```

Comment:
I think "gagent_install_cmd"  can be got in the function, not need to be a param.

Reply:
agree with@kimi1978wy , as the function name is `gagent_install`, I would suggest omit `gagent_install_cmd` for the function, use it directly in the function body.
The same to the `gagent_uninstall`.

Reply:
thanks you two,  delete this param.
pls have a check @hereischen @kimi1978wy